intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -1137,11 +1137,11 @@ \subsection{Activating and de-activating modules}
 
 \ifleuven
 \begin{prompt}
-%\shellcmd{module load matlab Python/2.7.6-foss-2014a}%
+%\shellcmd{module load matlab/2018a Python/2.7.6-foss-2014a}%
 \end{prompt}
 \else
 \begin{prompt}
-%\shellcmd{module load MATLAB Python/2.7.9-intel-2015a}%
+%\shellcmd{module load MATLAB/2018a Python/2.7.9-intel-2015a}%
```

Comment:
`Python/2.7.9-intel-2015a` is ancient, please update to `Python/2.7.14-intel-2018a`

Not sure what to use for KUL there (so leave as I guess)